tox.ini

```diff
@@ -30,4 +30,4 @@ max-line-length = 120
 import-order-style = google
 # Ignoring requirements of the docstrings in public packages, classes, functions, etc.
 ignore = D100, D101, D102, D103, D104, D105, D106, D107
-application-import-names = module, api, bridge_lti
```

Comment:
I see why did you add this line, but I prefer to use the whole path to module e.g. ```from module.models import Collection``` or ```from module.views import SomeView```.

Also, I had some troubles in tests when I used local imports... 

but it's only my opinion and we could discuss it...

Reply:
It is not like you described, here modules are added for the config to let tox recognize internal project's modules.